app/uk/gov/hmrc/eoricommoncomponent/frontend/views/migration/how_can_we_identify_you_nino.scala.html

```diff
@@ -28,6 +28,11 @@
 
 @(ninoForm: Form[IdMatchModel], isInReviewMode: Boolean, submit: Call)(implicit request: Request[_], messages: Messages)
 
+@hintNino = {
```

Comment:
Where is this used? Can't see a reference

Reply:
unused hintNino has been now removed. Please review.